src/contrib/cluster/Akka.Cluster.Tools/PublishSubscribe/DistributedPubSubMediator.cs

```diff
@@ -183,12 +186,11 @@ public DistributedPubSubMediator(DistributedPubSubSettings settings)
             });
             Receive<Publish>(publish =>
             {
-                var topic = Uri.EscapeDataString(publish.Topic);
-                var path = Self.Path / topic;
+                string path = Internal.Utils.MakeKey(Self.Path / Internal.Utils.EncodeName(publish.Topic));
```

Comment:
I'm planning to do changes like you mentioned, right before Cluster.Tools become stable